### PR TITLE
match sub_80029D0

### PR DESCRIPTION
**Pull request by jiangzhengwenjz**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -98,7 +98,7 @@ $(ELF): $(OBJS) $(LDSCRIPT)
 # Build c sources, and ensure alignment
 $(C_BUILDDIR)/%.o : $(C_SUBDIR)/%.c
 	$(CPP) $(CPPFLAGS) $< | $(CC1) $(CC1FLAGS) -o $(C_BUILDDIR)/$*.s
-	@echo ".text\n\t.align\t2, 0\n" >> $(C_BUILDDIR)/$*.s
```
Comment (freshollie) [inline Makefile:101]: Is this a linux issue? This breaks the build on MacOS

Comment (jiangzhengwenjz) [reply Makefile:101]: The original one won't translate \n and \t into newline and tab in linux. I actually checked pokeemerald and they also used `-e` :/

**Comment by jiangzhengwenjz** [comment]
Anyway, I reverted the change in Makefile. It's not important in this PR.

**Review by freshollie — APPROVED** [review]
Thanks for this :)

**Pull request merged**